tests/bluetooth/bsim_bt/bsim_test_multiple/prj.conf

```diff
@@ -10,7 +10,7 @@ CONFIG_BT_AUTO_PHY_UPDATE=y
 CONFIG_BT_USER_DATA_LEN_UPDATE=y
 CONFIG_BT_AUTO_DATA_LEN_UPDATE=y
 
-CONFIG_BT_MAX_CONN=250
+CONFIG_BT_MAX_CONN=150
```

Comment:
The "new" LLCP doesn't support 250 connections? 
Should `CONFIG_BT_ID_MAX` be reduced as well then?

Reply:
No, we need to fix the new LLCP so that it supports 250 connections

Reply:
Tracked by: https://github.com/zephyrproject-rtos/zephyr/issues/48823, but is this a bug or a design limitation?

Reply:
I believe this is a bug